Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/ErrorHandlingTransmissionPolicyTest.cs

```diff
@@ -219,7 +285,7 @@ private class TestableErrorHandlingTransmissionPolicy : ErrorHandlingTransmissio
         {
             public TimeSpan BackOffTime { get; set; }
 
-            public override TimeSpan GetBackOffTime(NameValueCollection headers = null)
+            protected override TimeSpan GetBackOffTime(NameValueCollection headers = null)
```

Comment:
That is a test class that allows to override Backoff time in tests. Method is protected  because that is what we have in the ErrorHandlingTransmissionPolicy class. public property is public so we can easily set what we want in unit tests 

---
In reply to: [64498867](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/232#discussion_r64498867) [](ancestors = 64498867)